libc/src/time/clockGetTimeImpl.h

```diff
@@ -0,0 +1,47 @@
+//===- Linux implementation of the POSIX clock_gettime function -*- C++ -*-===//
```

Comment:
This is a linux implementation so should live in the `linux` directory. Else, LGTM.